examples/js/src/main.ts

```diff
@@ -0,0 +1,11 @@
+import { Amplify } from 'aws-amplify';
+import { Authenticator } from '@aws-amplify/ui-standalone';
+import awsExports from '../../../environments/auth-with-username-no-attributes/src/aws-exports';
+
+import '@aws-amplify/ui-react/styles.css';
```

Comment:
I think this should either be:
1. Style injected (even for the react package perhaps?)
2. Or be re-exported from the `ui-standalone` package. Since usage of `Authenticator` from the standalone package should not involve pulling in something from `ui-react`

Reply:
Re-export is the way we've been going 👍